packages/web/src/common/components/Table.tsx

```diff
@@ -11,14 +13,13 @@ interface TableProps<T> {
   emptyMessage?: string;
   count?: number;
   footer?: React.ReactNode;
+  rowLink?: (row: T) => string | { pathname: string } | undefined;
```

Comment:
optional이니까 일케 해도 되지 않나용?
```suggestion
  rowLink?: (row: T) => string | { pathname: string };
```